src/modules/auth/dto/login.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Transform } from 'class-transformer';
+import { IsEmail, IsString, MinLength } from 'class-validator';
+
+export class LoginDto {
```

Comment:
can u update this to LoginDTO i mean use the DTO sufix in upper case

Reply:
Right now